shared/reducers/config.js

```diff
@@ -8,6 +8,7 @@ import type {Config, GetCurrentStatusRes, ExtendedStatus} from '../constants/typ
 
 export type ConfigState = {
   globalError: ?Error,
+  daemonError: ?string,
```

Comment:
@gabriel Wanted us to move into using Error types consistently for error like things. I think that makes sense, so this should be an Error type too